pages/participantinfo.tsx

```diff
@@ -13,79 +13,143 @@ import {
+        <Stack spacing={8}>
+            <HStack spacing="24px">
+                <FormControl id="first-name">
+                    <FormLabel>First name</FormLabel>
+                    <Input placeholder="First name" />
+                </FormControl>
+                <FormControl id="last-name">
+                    <FormLabel>Last name</FormLabel>
+                    <Input placeholder="Last name" />
+                </FormControl>
+            </HStack>
+            <FormControl id="date-of-birth">
+                <FormLabel>Date Of Birth</FormLabel>
+                <Input placeholder="Date Of Birth" />
+            </FormControl>
+            <FormControl id="street-address">
+                <FormLabel>Street Address</FormLabel>
+                <Input placeholder="Street Address" />
+            </FormControl>
+            <HStack spacing="24px">
+                <FormControl id="postal-code">
+                    <FormLabel>Postal Code</FormLabel>
+                    <Input placeholder="Postal Code" />
+                </FormControl>
+                <FormControl id="city">
+                    <FormLabel>City</FormLabel>
+                    <Input placeholder="City" />
+                </FormControl>
+                <FormControl id="province">
+                    <FormLabel>Province</FormLabel>
+                    <Input placeholder="Province" />
+                </FormControl>
+            </HStack>
+            <FormControl id="school">
+                <FormLabel>School (if applicable)</FormLabel>
+                <Input placeholder="School (if applicable)" />
+            </FormControl>
+            <FormControl id="grade">
+                <FormLabel>Grade (if applicable)</FormLabel>
+                <Input placeholder="Grade (if applicable)" />
+            </FormControl>
+        </Stack>,
+        <Stack spacing={8}>
+            <Text>Sample page 2 - add form fields</Text>
+        </Stack>,
+        <Stack spacing={8}>
+            <Text>Sample page 3 - add form fields</Text>
+        </Stack>,
+        <Stack spacing={8}>
+            <Text>Sample page 4 - add form fields</Text>
+        </Stack>,
+        <Stack spacing={8}>
+            <Text>Sample page 5 - add form fields</Text>
+        </Stack>,
+        <Stack spacing={8}>
+            <Text>Sample page 6 - add form fields</Text>
+        </Stack>,
+        <Stack spacing={8}>
+            <Text>Sample page 7 - add form fields</Text>
+        </Stack>,
+        <Stack spacing={8}>
+            <Text>Done!</Text>
+        </Stack>,
+    ];
+
     return (
         <Center>
-            <Box width="1000px">
+            <Box>
+                <Button
+                    onClick={() =>
+                        setPageNum((prevPage) => Math.max(prevPage - 1, 0))
+                    }
+                >
+                    Back
+                </Button>
                 <Text fontWeight="700" fontSize="36px">
                     Participant Information
```

Comment:
@sohak22 see https://github.com/uwblueprint/social-diversity-for-children/pull/43/commits/75ad5f0f557f6a34d70c7412bd5f539ea017524e#r675278037